### PR TITLE
Client upload is functional

### PR DESCRIPTION
**Pull request by Dulinag**

### CODE REVIEW

**Review by henryfan03 — APPROVED** [review]
lgtm

**Comment by George-Sucuzhanay** [comment]
make sure to add a PR comment for the next PR's


ex:

Added a handler func to execute unsplash API call whether the backend server is running or not

Testing Instructions:
- Run npm start on front-end repo
- Navigate to unsplash page
- Insert a query into input box and submit a search query
- Ensure api call returns a response based on search query

File: client/components/upload.jsx [hunk 1]
```diff
@@ -34,6 +38,27 @@ function FileUpload() {
...
+  };
+
+  const handleFileInputChange = (event) => {
+    const files = event.target.files;
+    for (let i = 0; i < files.length; i++) {
+      addFile(files[i]);
+    }
+  };
+
+  const handleDelete = (url) => {
```
Comment (George-Sucuzhanay) [inline client/components/upload.jsx:55]: Can you clarify the action of handleDelete and cancel? Thanks!

**Review by George-Sucuzhanay — DISMISSED** [review]

File: client/components/upload.jsx (hunk 1 above)
Comment (Dulinag) [reply client/components/upload.jsx:55]: Handle delete image by image when trash bin is clicked. Handle cancel deletes all images on upload.

**Comment by George-Sucuzhanay** [comment]
Screen Shot 2024-03-11 at 11 56 55 AM
Screen Shot 2024-03-11 at 11 56 55 AM (2)

Thank you for clarifying it was hard to see because of the icon color and the images were quite small in my screen. It first glance I could not tell where they were due to the colors of the photos. Do you think we should clarify this?

**Comment by George-Sucuzhanay** [comment]
an highlighted x icon on the upper right corner or a background color behind the icon could work as well
Screen Shot 2024-03-11 at 12 00 07 PM
Screen Shot 2024-03-11 at 12 00 07 PM (2)

**Comment by Dulinag** [comment]
Size of pictures can be fixed but the color scheme seems to be good for now. Need input from all of team before I changed colors or design.

**Comment by EricLi-Dev** [comment]
Did we want to change the color scheme here? I agree with the image size, think it'd be good if we can get them a bit bigger

**Review by EricLi-Dev — APPROVED** [review]
looks good!

**Review by jubuyer — APPROVED** [review]
Looks good to merge

**Pull request merged**